code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -517,10 +513,12 @@
 		var/mob/living/simple_animal/hostile/H = M
 		H.AIStatus = initial(H.AIStatus)
 
-
 /obj/effect/timestop/wizard
 	duration = 100
 
+/obj/effect/timestop/wizard/New()
+	..()
+	timestop()
```

Comment:
Don't I have to ..() to call all the stuff from normal New, which in turn would call the timestop() if I added it to the New of the basetype?